modules/icons/runtime/components/nuxt-icon.vue

```diff
@@ -30,7 +30,7 @@ async function getIcon() {
   }
 }
 
-await getIcon();
+getIcon();
```

Comment:
`getIcon` is an async function and should be called with an await, or the possible error should be handled otherwise. Leaving it like this is negligent IMO.

Reply:
Nuxt handles the error but it's a nasty fullscreen error. Moved the try/catch and replaced it with console.warn instead of throwing 👍 

But awaiting in `setup` is not fun, as the component becomes an async component, and you start needing <suspense> and stuff.

Awaiting here causes the icon component to not be rendered the first few frames, and that causes layout shifting when it finally appears (especially noticable inside dropdowns). It's better to not await and render a placeholder box the same size as the icon instead.